### PR TITLE
Fisrt PR:  Hello Microverse

### PR DESCRIPTION
**Pull request by gdvl777**
Create branches with different base branches.
Understand the concept of version control.
Follow GitHub flow.
Open pull requests in GitHub.
Write descriptive, easy to understand commit messages.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (Qoosim) [inline .gitignore]: - [x] It is required that test.md file should be created and ignore it by adding it to this file. Kindly create the file and it to this file in order to ignre it from appearing on your repository.


### Project Requirement
test-file

File: LICENCE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (Qoosim) [inline LICENCE.md]: - [x] Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set up the MIT License for your project. The link is broken at the moment.

license

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```
Comment (Qoosim) [inline index.html]: - [x] It is required to give a class name to your `h1` element and use that class name inside your style.css to style the content of the `h1` tag.

style

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct . :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

- [ ] It is a good practice to make your PR title descriptive. Something like `Hello Microverse` would have been the best title for this pull request. Kindly change that.

pr-title


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,288 +1,233 @@
...
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+## 👥 Authors: GhostEsso
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+- GitHub: [@githubhandle](https://github.com/GhostEsso)
+- Twitter: [@twitterhandle](https://twitter.com/TambanaEssohana)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/essohanam-tambana-62aa081a1/?originalSubdomain=tg)
```
Comment (TEMESGEN-G25) [inline README.md:171]: [OPTONAL]
- Kindly, use the links of your own GitHub, Twitter, etc accounts on your README file.

**Review by TEMESGEN-G25 — APPROVED** [review]
Hi @gdvl777, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
:heavy_check_mark: Great job setting up linters.
:heavy_check_mark: Nice job on adding the test.md file into .gitignore.

## Optional suggestions
- I recommend not marking any of your conversations as resolved. That way other code reviewers won't have to click to see the required changes set by the previous reviewer.

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**